ParseStarterProject/AppDelegate.swift

```diff
@@ -45,7 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // PFFacebookUtils.initializeFacebook()
         // ****************************************************************************
 
-        PFUser.enableAutomaticUser()
+        // PFUser.enableAutomaticUser()
```

Comment:
Needed to get rid of this line. It allowed for anonymous login (so the user is always logged in)